src/bloomf.ml

```diff
@@ -78,6 +78,75 @@ let size_estimate t =
   let xf = float_of_int (Bitv.pop t.b) in
   int_of_float (-.mf /. kf *. log (1. -. (xf /. mf)))
 
+let to_buf t =
+  let gs = String.make 1 (Char.chr 29) in
+  let rs = String.make 1 (Char.chr 30) in
+  let us = String.make 1 (Char.chr 31) in
+  let buf =
+    Buffer.create (5 + (4 * List.length t.p_len) + (2 * Bitv.length t.b))
```

Comment:
How did you compute that size ? (also applies to other locations)

Reply:
This is a lower bound. According to the docs, the buffer will get expanded as necessary. This simply assumes that each element of an array is one byte and multiplies by that + the number of separators we're adding. Getting this "wrong" isn't a huge deal, but I wanted to start somewhere that wasn't zero.